cli/src/main/java/fr/pilato/elasticsearch/crawler/fs/cli/FsCrawler.java

```diff
@@ -261,13 +258,7 @@ public static void main(String[] args) throws Exception {
                 Path jobMappingDir = configDir.resolve(fsSettings.getName()).resolve("_mappings");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "jobMappingDir". ([squid:S1854](https://sonarcloud.io/organizations/dadoonet-github/rules#rule_key=squid%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.pilato.elasticsearch.crawler%3Afscrawler-parent&pullRequest=579&issues=AWTql2IgrFWWQMKkjhyI&open=AWTql2IgrFWWQMKkjhyI)

[ISSUE_KEY]: AWTql2IgrFWWQMKkjhyI